ballista/README.md

```diff
@@ -35,9 +35,30 @@ Ballista can be deployed as a standalone cluster and also supports [Kubernetes](
 case, the scheduler can be configured to use [etcd](https://etcd.io/) as a backing store to (eventually) provide
 redundancy in the case of a scheduler failing.
 
+# Getting Started
+
+Fully working examples are available. Refer to the [Ballista Examples README](../ballista-examples/README.md) for 
+more information.
+
+## Distributed Scheduler Overview
+
+Ballista uses the DataFusion query execution framework to create a physical plan and then transforms it into a 
+distributed physical plan by breaking the query down into stages whenever the partitioning scheme changes.
+
+Specifically, any `RepartitionExec` operatoris is replaced with an `UnresolvedShuffleExec` and the child operator 
```

Comment:
```suggestion
Specifically, any `RepartitionExec` operator is is replaced with an `UnresolvedShuffleExec` and the child operator 
```

Reply:
we have an extra is here right?

Reply:
oh man -- good eyes. You are right -- the correct suggestion is:

```suggestion
Specifically, any `RepartitionExec` operator is replaced with an `UnresolvedShuffleExec` and the child operator 
```